index.html

```diff
@@ -0,0 +1,81 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet noopener noreferrer" href="css/style.css" />
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
- [ ] It's a good idea to always keep your script insertion just before the closing body tag `</body>`. This will allow all variables defined within the Html code to preload before the javascript file is loaded.